Tests/Unit/Module/DocumentRule.tests.ps1

```diff
@@ -15,6 +15,23 @@ try
 
                 If unapproved shared accounts exist, this is a finding.'
             }
+            @{
+                Id = "V-7069"
+                Severity = "medium"
+                title="APPNET0055 CAS and Policy Config File Backups"
```

Comment:
there should be a space after title or before the equal sign.

Reply:
Updated